### ISSUE TITLE
feature: Workspace name appears on mouseover

### ISSUE THREAD

**Issue by grumphus** [webcatalog/webcatalog-legacy#442]
Hi! I really like the workspace feature and its ability to set a custom name. My only gripe is that you can only see the workspace names in the Window menu. It would be nice if the custom name of the workstation appears as well when you mouseover the sidebar buttons.

Thank you!

**Comment by quanglam2807** [comment]
Thanks for the feedback. This is currently blocked by https://github.com/electron/electron/issues/15899.

**Comment by edmundlaugasson** [comment]
It would  be even more better to see permanently workspace names - this should be configurable option - looks like it is already done via View -> Workspace size - the shortcut key would be in wide workspace size good to keep smaller under icon to save space. Currently shortcut key requires extra width and that is not good.
Still the problem is, that cannot change the navigation bar width manually. Currently there is fixed size, which is wasting space.

**Comment by quanglam2807** [comment]
The workspace name will now show up on mouseover (even though not immediately). I'll try to improve the speed in the future.
